_posts/2017-06-27-greenpithumb.md

```diff
@@ -211,7 +231,7 @@ It's a lesson I learned [long ago](https://www.joelonsoftware.com/2002/03/04/not
 
 ## Electronics: start with the basics
 
-{% include image.html file="arduino-starter-kit.jpg" alt="Arduino starter kit" max_width="300px" class="align-left" img_link="<http://amzn.to/2rpRLCf>" %}
+{% include image.html file="arduino-starter-kit.jpg" alt="Arduino starter kit" max_width="300px" class="align-left" link_url="<http://amzn.to/2rpRLCf>" %}
```

Comment:
Do we need the angle brackets or is it possible to just write the links as normal strings, e.g. `link_url="http://amzn.to/2rpRLCf"`?

Reply:
Ah, yeah, I forgot to call that issue out.  If you don't include the angle brackets, then the markdown linter will fail on [Rule 034: No bare URLs allowed](https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md).  So, I added the angle brackets to work around that issue and that's how you would do it within markdown as well...  I know that isn't ideal or intuitive though...

The only way around the rule that I can think of would be:
1. Add an exception for that linting rule, but I don't really like or recommend that option.
2. We could configure the `link_url` parameter to just accept the url without the protocol.  For example, `link_url="amzn.to/2rpRLCf"` and then build the link within the template adding the `https` protocol.  That would fool the linter into not flagging that as an issue.  The only issue would be if you ever specifically would need regular `http`.  For example, it looks like the above example uses `http` vs `https` although it looks like the result goes to `https` either way, so not sure it matters in this case.  But if you did ever link to a URL that needed regular `http`, we would have to add yet another parameter to signify protocol if that were the case, which then starts to get a little clumsy in my opinion, but we can do something like that if you prefer instead of the angle brackets?

Reply:
Oh, gotcha. You can turn off the linter rule. I don't use a lot of bare URLs in text, and I feel like when I do, I'll notice if Jekyll is not rendering it as a link.

Reply:
OK, that makes sense and that'll work!  The linter rule for bare urls is now turned off in f4675fd so you no longer need to put angle brackets in with the `link_url` param either.